app/javascript/components/pages/export-list/container.jsx

```diff
@@ -55,6 +55,7 @@ const ExportList = () => {
                       {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
                       <div
                         className={css.link}
+                        data-testid="offline"
```

Comment:
Can this be moved to the `DisableOffline` component?

Reply:
Done